t/14-isotovideo.t

```diff
@@ -186,6 +186,21 @@ subtest 'error handling when loading test schedule' => sub {
     };
 };
 
+subtest 'load test success when there is no PRODUCTDIR' => sub {
+    chdir($pool_dir);
+    path(bmwqemu::STATE_FILE)->remove if -e bmwqemu::STATE_FILE;
+    path('vars.json')->remove         if -e 'vars.json';
+    mkdir('opensuse')                                                   unless -e 'opensuse';
```

Comment:
I suggest to avoid the "opensuse" reference completely here and use something like `my_cases`

Reply:
fixed